### ISSUE TITLE
Send inline bot query result using keyboard (Up/Down/Enter)

### ISSUE THREAD

**Issue by alekseyl1992** [telegramdesktop/tdesktop#1470]
It would be really awesome if one can select and send inline bot's query result using keyboard only.
For example: using Up/Down arrows to select result and Enter to send.
First result should be selected by default.

**Comment by telegramdesktop** [comment]
@alekseyl1992 Yes, I thought about that, perhaps in some future version..

**Comment by igorsantos07** [comment]
+1 on this. My main upvote on Telegram Desktop versus any other messaging client is the freaking awesome support for keyboard navigation. That's definitely missing! :(

**Comment by DaFri-Nochiterov** [comment]
+1
And... Browse links, commands, e-mails, tags, names of channels and users by *\<Tab\>*.

| Links and usernames | Username |
|:---:|:---:|
| ![image](https://cloud.githubusercontent.com/assets/10401817/13067440/30834498-d49a-11e5-9401-7ed22e4a0043.png) | ![image](https://cloud.githubusercontent.com/assets/10401817/13067442/375df0ce-d49a-11e5-96a4-82cfac1669f1.png) |
|**Tags and channel name**| **E-mail, link, command, username** |
| ![image](https://cloud.githubusercontent.com/assets/10401817/13071841/33b91414-d4c0-11e5-85f0-6c9dfb24bf3c.png) | ![image](https://cloud.githubusercontent.com/assets/10401817/13071925/b77a9cdc-d4c0-11e5-849b-608ff5228446.png) |

**Comment by stek29** [comment]
Bump to protect from auto close.

**Comment by SmartManoj** [comment]
> +1
> And... Browse links, commands, e-mails, tags, names of channels and users by .
> 
> Links and usernames	Username
> ![image](https://cloud.githubusercontent.com/assets/10401817/13067440/30834498-d49a-11e5-9401-7ed22e4a0043.png)	![image](https://cloud.githubusercontent.com/assets/10401817/13067442/375df0ce-d49a-11e5-96a4-82cfac1669f1.png)
> **Tags and channel name**	**E-mail, link, command, username**
> ![image](https://cloud.githubusercontent.com/assets/10401817/13071841/33b91414-d4c0-11e5-85f0-6c9dfb24bf3c.png)	![image](https://cloud.githubusercontent.com/assets/10401817/13071925/b77a9cdc-d4c0-11e5-849b-608ff5228446.png)

What is this?

**Comment by SmartManoj** [comment]
Also Ctrl +{numbering} to select particular one

**Comment by SmartManoj** [comment]
@patgod85 https://github.com/probot/ideas/issues/75
